rts/Game/UI/KeySet.cpp

```diff
@@ -193,11 +211,11 @@ bool CKeySet::Parse(const std::string& token, bool showerror)
 		return false;
 	}
 
-	if (IsModifier())
```

Comment:
This made absolutely no sense, I have no idea why this was ever a case.

Perhaps a previous 'lets be safe' case originating from statefulCommands handling like in KeyBindings.cpp (that still remains the case) that didn't get removed.

Reply:
Is the behaviour for non-modifier-only bindings with Any still the same? Say things like `Any+Ctrl+X`.

Reply:
This is a more "no idea why this exists, there's not a reasonable explanation for this" than "no idea why this exists, I don't understand what this does, lets remove it".

The only cases that will work differently are ones that rely on pure modifier bindings, e.g. `bind something [mod\+*]?mod`, the difference being that they wont be equivalent to `any+mod` anymore, which is an improvement.

No games rely on this unexpected behavior as far as I have investigated.